### ISSUE TITLE
Crash with dank/null

### ISSUE THREAD

**Issue by Grovert11** [RS485/LogisticsPipes#1395]
The Minecraft instance crashed when I put my dank/null in a dock with a chassis pipe with a quicksort module attached to it. Just updated LP to 212.

```
java.lang.UnsupportedOperationException: Couldn't extract the items already sent from the inventory
	at logisticspipes.modules.ModuleQuickSort.tick(ModuleQuickSort.java:230)
	at logisticspipes.modules.ChassiModule.tick(ChassiModule.java:137)
	at logisticspipes.pipes.basic.CoreRoutedPipe.updateEntity(CoreRoutedPipe.java:444)
	at logisticspipes.pipes.basic.LogisticsTileGenericPipe.func_73660_a(LogisticsTileGenericPipe.java:213)
	at net.minecraft.world.World.func_72939_s(World.java:1838)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:613)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:767)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:185)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by OmaiGrundles** [comment]
Dank null specifically prevents pushing items into it via pipes and such, but allows things to pull out of it, due to balance reasons I believe. Since a sorting function would try to move items around...

**Comment by hron84** [comment]
Is there any capability LP could check if the inventory can accept an item? Maybe this issue is on DankNull side because the dock should refuse to accept an item.

**Comment by Grovert11** [comment]
But the quicksort module should not be insuring items right? It is just for extracting items that have a valid destination in the LP network

**Comment by hron84** [comment]
It depends on. AFAIK if there is no place for the extracted item, LP could try to insert back and that could cause an issue.

**Comment by bziemons** [comment]
Fixed in 86015c19e41781cd92c80b0a5b72946ca344a450
